okhttp/src/main/java/okhttp3/internal/http/RetryAndFollowUpInterceptor.java

```diff
@@ -70,13 +71,12 @@ public RetryAndFollowUpInterceptor(OkHttpClient client) {
     RealInterceptorChain realChain = (RealInterceptorChain) chain;
     Transmitter transmitter = realChain.transmitter();
 
-    transmitter.newStreamAllocation(request);
```

Comment:
This PR has 2 interesting parts.

1. Replacing newStreamAllocation() with prepareToConnect(). The difference is prepareToConnect() works if there’s another stream already ready.
2. Moving the releaseStreamAllocation() calls out of here and into RealCall.